gbfs.md

```diff
@@ -1128,22 +1128,20 @@ Field Name | REQUIRED | Type | Defines
 `plans` | Yes | Array&lt;Object&gt; | Contains one object per pricing plan.
 `plans[].plan_id` | Yes | ID | Identifier for a pricing plan in the system.
 `plans[].url` | OPTIONAL | URL | URL where the customer can learn more about this pricing plan.
-`plans[].name` <br/>*(as of v3.0-RC)* | Yes | Array&lt;Localized String&gt; | Name of this pricing plan.
+`plans[].name` <br/>*(as of v3.0)* | Yes | Array&lt;Localized String&gt; | Name of this pricing plan.
 `plans[].currency` | Yes | String | Currency used to pay the fare. <br /><br /> This pricing is in ISO 4217 code: http://en.wikipedia.org/wiki/ISO_4217 <br />(for example, `CAD` for Canadian dollars, `EUR` for euros, or `JPY` for Japanese yen.)
 `plans[].price` | Yes | Non-Negative Float | Fare price, in the unit specified by `currency`. <br/>*(added in v2.2)* In case of non-rate price, this field is the total price. In case of rate price, this field is the base price that is charged only once per trip (typically the price for unlocking) in addition to `per_km_pricing` and/or `per_min_pricing`.
-`plans[].reservation_price_per_min` <br/>*(added in v3.0-RC)*  | OPTIONAL | Non-Negative Float | The cost, described as per minute rate, to reserve the vehicle prior to beginning a rental. This amount is charged for each minute of the vehicle reservation until the rental is initiated, or until the number of minutes defined in `vehicle_types.json#default_reserve_time` elapses, whichever comes first. When using this field, you MUST declare a value in `vehicle_types.json#default_reserve_time`. This field MUST NOT be combined in a single pricing plan with `reservation_price_flat_rate`.
```

Comment:
These are the ones that didn't make it, right? 
Do we want to somehow keep them somewhere (e. g. start a new RC automatically) or they go back to the issue stage?

Reply:
Exactly. https://github.com/MobilityData/gbfs/pull/546 and https://github.com/MobilityData/gbfs/pull/457 are delayed to the next version as no producer came forward to implement these changes since they were voted.

Reply:
They will be added to the next Release Candidate versions according to the [Version Release Cycles](https://github.com/MobilityData/gbfs/blob/master/governance.md#version-release-cycles):

Date | Version | Changes to add
-- | -- | --
May 2024 | v3.1 (MINOR version) | https://github.com/MobilityData/gbfs/pull/457 (non-breaking change)
November 2024 | v4.0 (MAJOR version) | https://github.com/MobilityData/gbfs/pull/546 (breaking change)